### ISSUE TITLE
Replace "chosen" script with a more accessible option

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7722]
## Enhancement Request

> ...we should look at deprecating Chosen (which itself has been Deprecated https://github.com/harvesthq/chosen) and replacing it with a more maintained alternative, such as:
> 
> - https://github.com/jshjohnson/Choices
> - https://github.com/woocommerce/selectwoo
> 
> Related https://github.com/easydigitaldownloads/easy-digital-downloads/issues/2758 https://github.com/easydigitaldownloads/easy-digital-downloads/issues/4375
> 
> _Originally posted by @spencerfinnell in https://github.com/easydigitaldownloads/easy-digital-downloads/issues/7583#issuecomment-587152955_

### Justification or use case
Any select option using Chosen is currently not accessible to keyboard or screen reader users.

**Comment by spencerfinnell** [comment]
Noting that @robincornett opened this issue as a place to discuss alternatives/replacements for Chosen and to keep #7583 as a tracking issue for removing existing unneeded usage.

**Comment by cklosowski** [comment]
Is Select2 an option? It seems to still be maintained.

**Comment by robincornett** [comment]
selectWoo is a select2 fork with accessibility related improvements and fixes so is probably a better option.

**Comment by cklosowski** [comment]
I find it interesting that they chose to fork instead of contribute upstream. Maybe the Select2 team isn’t open to their suggestions.

**Comment by JJJ** [comment]
I spent last night forking Chosen, here: https://github.com/JJJ/chosen

* 25 pull requests merged (including Aria & Role support!)
* Oldest pull request was 8 years old (yikes!)
* It's fully accessible (probably!)
* It's working (tests are failing tho!)
* Styling modernized (kinda!)

Please consider giving my fork a try. It is drop-in replacement for what is in EDD already, and I'd love some help getting the tests to pass 🤗

**Comment by spencerfinnell** [comment]
I'm not sure we should close this out completely until Chosen adds mobile support: https://github.com/JJJ/chosen/issues/33

Currently searching for Downloads (or anything dynamic) does not work on mobile.

**Comment by robincornett** [comment]
Would it be worth opening a new issue, @spencerfinnell? I saw that John updated Chosen to 2.2.0 and what we've put into EDD 3.0 is 2.1.0, so an update would be needed anyway. Also, based on your comments there, this is a current issue with 2.9.x, yes?

**Comment by spencerfinnell** [comment]
@robincornett Sure -- probably a good idea to track it separately.
